cognite/client/data_classes/data_modeling/views.py

```diff
@@ -428,6 +429,7 @@ def as_apply(self) -> SingleHopConnectionDefinitionApply:
             description=self.description,
             edge_source=self.edge_source,
             direction=self.direction,
+            connection_type=self.connection_type,
```

Comment:
connection_type can just be removed from the constructor of this class. It's implicitly always `multiEdgeConnection` for this class.

Reply:
So it is more like a class variable?